raml/src/main/kotlin/de/codecentric/hikaku/converters/raml/RamlConverter.kt

```diff
@@ -68,7 +69,9 @@ class RamlConverter(private val ramlSpecification: File) : AbstractEndpointConve
                             pathParameters = it.resource()?.hikakuPathParameters().orEmpty(),
                             headerParameters = it?.hikakuHeaderParameters().orEmpty(),
                             consumes = it.requestMediaTypes(),
-                            produces = it.responseMediaTypes()
+                            produces = it.responseMediaTypes(),
+                            deprecated = it.annotations().any { i -> i.annotation().name() == "deprecated" }
```

Comment:
I think it would be nice to extract that into a `isEndpointDeprecated` function.